packages/react/src/checkout/draftOrders/hooks/useRemoveDraftOrders.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  getRemoveDraftOrderError,
+  isRemoveDraftOrderLoading,
+  removeDraftOrder,
+  removeDraftOrderItem,
+  type StoreState,
+} from '@farfetch/blackout-redux';
+import { useSelector } from 'react-redux';
+import useAction from '../../../helpers/useAction.js';
+import type { DraftOrder } from '@farfetch/blackout-client';
+
+const useRemoveDraftOrders = (draftOrderId: DraftOrder['id']) => {
+  const fetchRemoveDraftOrderItem = useAction(removeDraftOrderItem);
```

Comment:
To have naming consistency, we should rename these variables. 
For actions should be `removeDraftOrderItemAction` and `removeDraftOrderAction`

take a look into the `useCheckout` hook